contrib/nydusd-http-server/README.md

```diff
@@ -0,0 +1,105 @@
+# Http server for nydusd
+A http server based on Rocket to serve static file for nydusd.
+
+
+## APIs supported
+support the following APIs:
+```bash
+HEAD /$namespace/$repo/blobs/sha256:xxx #get blob size
```

Comment:
This is API description does not have many details. At least give a definition for response when status code is 200/204？
What is important is to tell what is functionality of the server and when to and how to use it to solve what kind of problems.